webapp/src/editortoolbar.tsx

```diff
@@ -221,20 +226,14 @@ export class EditorToolbar extends data.Component<ISettingsProps, {}> {
     }
 
     renderCore() {
-        const { home, tutorialOptions, hideEditorFloats, collapseEditorTools, projectName, compiling, isSaving, simState, debugging, header } = this.props.parent.state;
-
-        if (home) return <div />; // Don't render if we're in the home screen
```

Comment:
why remove this?

Reply:
The component is not shown in app.tsx when in the home screen (it's toggled by `showEditorToolbar` which is false if `inHome`)